tests/FrontRow/App/HttpSpec.hs

```diff
@@ -0,0 +1,36 @@
+module FrontRow.App.HttpSpec
+  ( spec
+  ) where
+
+import Prelude
+
+import Control.Lens ((^?))
+import Data.Aeson
+import Data.Aeson.Lens
+import Data.Either (isLeft)
+import Data.Text (Text)
+import FrontRow.App.Http
+import Network.HTTP.Types.Status (status200)
+import Test.Hspec
+
+spec :: Spec
+spec = do
+  describe "httpJson" $ do
+    it "fetches JSON via HTTP" $ do
+      resp <- httpJson @_ @Value
```

Comment:
Interesting - so this means we let type inference choose the type of `m`, but we are explicitly setting the type of `a`?

Reply:
Yup!